src/app/contextBroker/contextBroker.cpp

```diff
@@ -86,10 +86,14 @@
 #include "common/globals.h"
```

Comment:
CNR entry missing.

cli.md should be modified to explain the new notification mode (a new bullet in the `-notificationMode` list)